en/app.ftl

```diff
@@ -300,3 +300,17 @@ survey-option-not-likely = Not likely
 vpn-promo-headline = Save 50% with a full year subscription
 vpn-promo-copy = Protect your online data and choose a VPN subscription plan that works for you.
 vpn-promo-cta = Get { -brand-name-mozilla-vpn }
+
+## Email wrapping (header and footer messages wrapped around forwarded emails)
+
+# Variables:
+#   $display_email (string) - This is the relay email address. Example: abc123@relay.firefox.com
+#   $linked_origin (html) - This is a link to the website, wrapped with inline styles for email. Example: <a href="https://relay.firefox.com" ...>Firefox Relay</a>
+forwarded-email-header-from = This message was forwarded from { $display_email } by { $linked_origin }.
+# Variables:
+#   $faq_link (html) - This is a link to the website FAQ, wrapped with inline styles for email. Example: <a href="https://relay.firefox.com/faq" ...>{nav-faq}</a>
+forwarded-email-header-attachment = Firefox Relay supports email forwarding (including attachments) of email up to 150KB in size. To learn more check our { $faq_link }. 
+forwarded-email-footer-manage-settings-text = manage settings for all aliases here.
```

Comment:
Should it be capital M in `Manage settings ...`?

Reply:
Hmm ... the full English text is "Stop email forwarding and manage settings for all aliases here." But the "manage settings for all aliases here" is a link. It might be easiest to just make the whole thing the link, so it can be translated as a single message.

Reply:
From the sequence of the strings, the first part of the sentence is not immediately before this one. If you are reusing the same string for different context, don't do that. It is possible to have a link to part of the string, right?

Reply:
Simplified this into a single `forwarded-email-footer` message and made the whole thing the link.